runner/cf-driver/prepare.sh

```diff
@@ -38,6 +38,77 @@ start_container () {
        --vars-file "$TMPVARFILE"
 }
 
+start_service () {
+    alias_name="$1"
+    container_id="$2"
+    image_name="$3"
+    container_entrypoint="$4"
+    container_command="$5"
+
+    if [ -z "$container_id" ] || [ -z "$image_name" ]; then
+       echo 'Usage: start_service CONTAINER_ID IMAGE_NAME CONTAINER_ENTRYPOINT CONTAINER_COMMAND'
+       exit 1
+    fi
+    if [ -n "$container_entrypoint" ] || [ -n "$container_command" ]; then
+        # TODO - cf push allows use of -c or --start-command but not a separate
+        # entrypoint. May need to add logic to gracefully convert entrypoint to
+        # a command.
+        echo '[cf-driver] container_entrypoint and container_command are not yet supported in services - Sorry!'
+        exit 1
+    fi
+
+    if cf app --guid "$container_id" >/dev/null 2>/dev/null ; then
+        echo '[cf-driver] Found old instance of runner service, deleting'
+        cf delete "$container_id"
+    fi
+
+    # TODO - Figure out how to handle command and non-global memory definition
+    cf push "$container_id" --docker-image "$image_name" -m "$WORKER_MEMORY" \
+        --no-route --health-check-type process
+
+    cf map-route "$container_id" apps.internal --hostname "$container_id"
+}
+
+allow_access_to_service () {
+    source_app="$1"
+    destination_service_app="$2"
+    current_org=$(echo "$VCAP_APPLICATION" | jq --raw-output ".organization_name")
+    current_space=$(echo "$VCAP_APPLICATION" | jq --raw-output ".space_name")
+
+    # TODO NOTE: This is foolish and allows all TCP ports for now.
```

Comment:
Perhaps add a:

~~~
echo '[cf-driver] [WARN] All TCP ports open. For testing only
~~~

Reply:
🤔 I am going to decline this and lean on https://github.com/GSA-TTS/gitlab-runner-cloudgov/issues/6 to close the loop.